### PR TITLE
Remove containerd e2e and node e2e duplicate jobs

### PR DESCRIPTION
**Pull request by adisky**
These jobs are duplicate
ci-containerd-e2e-ubuntu-gce == ci-cri-containerd-e2e-ubuntu-gce
ci-containerd-node-e2e == ci-cri-containerd-node-e2e

Signed-off-by: Aditi Sharma <adi.sky17@gmail.com>

### CODE REVIEW

**Comment by endocrimes** [comment]
We have a similar dupe with ci-cri-containerd-node-e2e/ci-containerd-node-e2e. wanna drop ci-cri-containerd-node-e2e too?

**Review by endocrimes — APPROVED** [review]
/lgtm

File: config/testgrids/config.yaml [hunk 1]
```diff
@@ -5,14 +5,6 @@
 # See the OWNERS docs at https://go.k8s.io/owners for more info
 
 # Start testgroups
-test_groups:
```
Comment (endocrimes) [inline config/testgrids/config.yaml:8]: need to keep this line

Comment (adisky) [reply config/testgrids/config.yaml:8]: oops, multitasking effect

**Review by SergeyKanzhelev — APPROVED** [review]

**Comment by adisky** [comment]
/assign @BenTheElder @spiffxp @wojtek-t 
(for testgrid approval)

**Pull request merged**